packages/ast-spec/tests/fixtures.test.ts

```diff
@@ -77,7 +77,7 @@ const FIXTURES: readonly Fixture[] = [...VALID_FIXTURES, ...ERROR_FIXTURES].map(
       isError: /[\\/]_error_[\\/]/.test(absolute),
       isJSX: ext.endsWith('x'),
       name,
-      relative: path.relative(SRC_DIR, absolute).replace(/\\/g, '/'),
+      relative: path.relative(SRC_DIR, absolute).replaceAll('\\', '/'),
```

Comment:
🤔 this new version is more characters and IMO equally clear. I'm -0.1.

Reply:
In this specific instance, or in the whole PR?

Reply:
I'm +1. Regex escaping is too footguny. We should prioritize string replacement whenever possible.

Reply:
I meant kind of halfway between this instance & the whole PR. This instance and the ones like it...

...but yeah, Josh brings up a good point on the footguns. I'm ok with the average being +0.9.